src/core/config.go

```diff
@@ -370,7 +370,7 @@ func DefaultConfiguration() *Configuration {
 	config.Go.EmbedTool = "//_please:please_go_embed"
 	config.Python.PexTool = "//_please:please_pex"
 	config.Java.JavacWorker = "//_please:javac_worker"
-	config.Java.JarCatTool = "//_please:jarcat"
+	config.Java.JarCatTool = "//_please:arcat"
```

Comment:
Should line 525 say "Defaults to arcat"?

Reply:
Good spot, thanks.

All those comments are actually out of date (they don't default to being in the install directory any more)...